pygmt/datasets/samples.py

```diff
@@ -331,7 +330,7 @@ def load_sample_data(name):
     >>> from pprint import pprint
     >>> from pygmt.datasets import list_sample_data, load_sample_data
     >>> # use list_sample_data to see the available datasets
-    >>> pprint(list_sample_data(), width=120)  # noqa: W505
+    >>> pprint(list_sample_data(), width=120)
```

Comment:
I think we need to add this back.

Reply:
Done in b36981162654b2c440b7ba71bb5c34f8b3a9dd3c